### PR TITLE
Spearbit Audit Changes

### PR DESCRIPTION
**Pull request by hexonaut**
Changes:

 * Don't silently cast the gas limit in Optimism

### CODE REVIEW

File: src/forwarders/OptimismForwarder.sol [hunk 1]
```diff
@@ -15,24 +15,24 @@ library OptimismForwarder {
         address l1CrossDomain,
         address target,
         bytes memory message,
-        uint256 gasLimit
+        uint32 gasLimit
```
Comment (lucas-manuel) [inline src/forwarders/OptimismForwarder.sol:18]: Original comment [here](https://github.com/spearbit-audits/review-makerdao/pull/1#discussion_r1724902322)

**Review by lucas-manuel — APPROVED** [review]
Approving pending spearbit responds indicating that its complete and don't want issue-based PRs

**Review by barrutko — APPROVED** [review]
LGTM

**Comment by lucas-manuel** [comment]
Addresses the following Spearbit issues:

https://github.com/spearbit-audits/review-makerdao/issues/20

**Pull request merged**